glide-core/redis-rs/redis/src/cluster_routing.rs

```diff
@@ -297,38 +415,77 @@ fn get_route(is_readonly: bool, key: &[u8]) -> Route {
 ///
 /// If all keys are routed to the same slot, there's no need to split the command,
 /// so a single node routing info will be returned.
+///
+/// # Arguments
+/// * `routable` - The command or structure containing key-related data that can be routed.
+/// * `cmd` - A byte slice representing the command name or opcode (e.g., `b"MGET"`).
+/// * `first_key_index` - The starting index in the command where the first key is located.
+/// * `args_pattern` - Specifies how keys and values are patterned in the command (e.g., `OnlyKeys`, `KeyValuePairs`).
+///
+/// # Returns
+/// `Some(RoutingInfo)` if routing info is created, indicating the command targets multiple slots or a single slot;
+/// `None` if no routing info could be derived.
```

Comment:
What about the atomicity of the operation? i.e. it can work in one node but failed in the other. Is this documented somewhere?

From `MSET` documentation:

> MSET is atomic, so all given keys are set at once. It is not possible for clients to see that some of the keys were updated while others are unchanged

Reply:
The document wasn't good enough, I updated the documentation for all multi-slot commands (MSET, MGET, DEL, EXISTS, TOUCH, WATCH, UNLINK) in java, python, node and go in commit:
https://github.com/valkey-io/valkey-glide/pull/2587/commits/b25442457f1ef7faa5256529ae3452cc8c2d4be5

@shohamazon @BoazBD @Yury-Fridlyand make sure to add this note to JSON.MGET when added.